worlds/oot_soh/location_access/dungeons/forest_temple.py

```diff
@@ -2,8 +2,465 @@
 
 if TYPE_CHECKING:
     from ... import SohWorld
-
+    
+class EventLocations(str, Enum):
+    FOREST_TEMPLE_MEG = "Forest Temple Meg"
+    FOREST_TEMPLE_JOELLE = "Forest Temple Joelle"
+    FOREST_TEMPLE_AMY = "Forest Temple Amy"
+    FOREST_TEMPLE_BETH = "Forest Temple Beth"
+    FOREST_TEMPLE_LOWER_STALFOS_FAIRY_POT = "Forest Temple Lower Stalfos Fairy Pot"
+    FOREST_TEMPLE_DEKU_BABA_STICKS = "Forest Temple Deku Baba Sticks"
+    FOREST_TEMPLE_DEKU_BABA_NUTS = "Forest Temple Deku Baba Nuts"
+    FOREST_TEMPLE_CLEAR = "Forest Temple Clear"
+    FOREST_TEMPLE_NE_OUTDOORS_UPPER_DRAIN_SEWER = "Forest Temple NE Outdoors Upper Drain Sewer"
```

Comment:
`FOREST_TEMPLE_CLEAR` exists as `CLEARED_FOREST_TEMPLE` in the global events, please use that one instead.